op-e2e/actions/dencun_fork_test.go

```diff
@@ -103,7 +103,7 @@ func TestDencunL1ForkAtGenesis(gt *testing.T) {
 	require.Equal(t, sequencer.SyncStatus().UnsafeL2, verifier.SyncStatus().UnsafeL2, "verifier and sequencer agree")
 }
 
-func TestDencunL2ForkAfterGenesis(gt *testing.T) {
+func TestDencunL2ForkAtGenesis(gt *testing.T) {
```

Comment:
I notice we are swapping the names of these tests, "AtGenesis" becomes "AfterGenesis" and vice-versa. But looking a the content of the tests, I'm not sure if that was intentional?

What is now called "TestDencunL2ForkAtGenesis" references "crossing the fork boundary" and contains 1 extra call to the miner than "TestDencunL2ForkAfterGenesis"

Reply:
Thanks! I swapped them as the AfterGenesis `offset` was 0 and the AtGenesis was 24 -- which seemed the opposite way around. 

Just pushed a commit to rework (and relocate) these tests.